x-pack/plugins/ml/public/application/overview/components/anomaly_detection_panel/actions.tsx

```diff
@@ -23,12 +24,31 @@ export const ExplorerLink: FC<Props> = ({ jobsList }) => {
       values: { jobsCount: jobsList.length, jobId: jobsList[0] && jobsList[0].id },
     }
   );
-  const { createLinkWithUserDefaults } = useCreateADLinks();
+  const [explorerPath, setExplorerPath] = useState<string>();
 
-  return (
+  const mlLocator = useMlLocator();
+
+  useEffect(
+    function generateExplorerUrl() {
+      mlLocator!
+        .getUrl({
+          page: ML_PAGES.ANOMALY_EXPLORER,
```

Comment:
If you open the results view in a separate tab / window with the browser 'Open link...' action, the time range in the view isn't getting passed to the Anomaly Explorer e.g.
```
https://localhost:5601/abc/app/ml/explorer?_g=(ml%3A(jobIds%3A!(apm_abnormal_trace_durations_nodejs%2Capm_decreased_throughput_nodejs)))&_a=(explorer%3A(mlExplorerFilter%3A()%2CmlExplorerSwimlane%3A()))
```

Reply:
Done in 1793db937e6b8db69331f0c0ea88326e3e9f1e4e